src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs

```diff
@@ -106,7 +110,8 @@ public void ListOfNonNull_containing_null_should_bubble_up_the_null()
             var data = new Data {ListOfStrings = new List<string> {"text", null, null}};
             var errors = new[]
             {
-                new ExecutionError("Error trying to resolve listOfNonNullable.")
+                new ExecutionError("Error trying to resolve listOfNonNullable.", new InvalidOperationException(
+                    "Cannot return a null member within a non-null list for list index 1."))
```

Comment:
There was no this message here before, but tests passed. This means that the test only checks the type of the inner exception or, in other words, the error code (INVALID_OPERATION, FORMAT, etc.). Let's add message validation for inner exceptions.

Reply:
😄 I read to the end of the file and saw that you have already done so. I would also add a check for matching exception types.

Reply:
No problem 👍 

Done.